src/Analyzers/MSTest.Analyzers/PublicMethodShouldBeTestMethodAnalyzer.cs

```diff
@@ -64,7 +64,9 @@ private static void AnalyzeSymbol(SymbolAnalysisContext context, INamedTypeSymbo
             return;
         }
 
-        if (!methodSymbol.HasValidTestMethodSignature(taskSymbol, valueTaskSymbol, canDiscoverInternals))
+        if (!methodSymbol.HasValidTestMethodSignature(taskSymbol, valueTaskSymbol, canDiscoverInternals)
+            || methodSymbol.IsVirtual
+            || methodSymbol.IsOverride)
```

Comment:
This is skipping virtual/override completely, is it intended?
Or do we want to walk the OverriddenMethod chain and skip only if something in the chain is an implementation of interface member?

Reply:
Let's ignore that for now.